test/e2e/space_test.go

```diff
@@ -0,0 +1,214 @@
+			VerifyToolchainStatus(t, hostAwait, memberAwait)
+		})
+	})
+
+	t.Run("verify MemberOperatorConfigs synced from ToolchainConfig to member clusters", func(t *testing.T) {
+		currentConfig := hostAwait.GetToolchainConfig()
+		expectedMemberConfiguration := currentConfig.Spec.Members.Default
+
+		t.Run("verify ToolchainConfig has synced status", func(t *testing.T) {
+			VerifyToolchainConfig(t, hostAwait, wait.UntilToolchainConfigHasSyncedStatus(ToolchainConfigSyncComplete()))
+		})
+		t.Run("verify MemberOperatorConfig was synced to member 1", func(t *testing.T) {
+			VerifyMemberOperatorConfig(t, hostAwait, memberAwait, wait.UntilMemberConfigMatches(expectedMemberConfiguration))
+		})
+		t.Run("verify MemberOperatorConfig was synced to member 2", func(t *testing.T) {
+			member2ExpectedConfig := testconfig.NewMemberOperatorConfigObj(testconfig.Webhook().Deploy(false))
+			VerifyMemberOperatorConfig(t, hostAwait, memberAwait2, wait.UntilMemberConfigMatches(member2ExpectedConfig.Spec))
+		})
+		t.Run("verify updated toolchainconfig is synced - go to unready", func(t *testing.T) {
+			// set the che required flag to true to force an error on the memberstatus (che is not installed in e2e test environments)
+			memberConfigurationWithCheRequired := testconfig.ModifyMemberOperatorConfigObj(memberAwait.GetMemberOperatorConfig(), testconfig.Che().Required(true))
+			hostAwait.UpdateToolchainConfig(testconfig.Members().Default(memberConfigurationWithCheRequired.Spec))
+
+			err := memberAwait.WaitForMemberStatus(
+				wait.UntilMemberStatusHasConditions(ToolchainStatusComponentsNotReady("[routes]")))
+			require.NoError(t, err, "failed while waiting for MemberStatus to contain error due to che being required")
+
+			_, err = hostAwait.WaitForToolchainStatus(
+				wait.UntilToolchainStatusHasConditions(ToolchainStatusComponentsNotReady("[members]"), ToolchainStatusUnreadyNotificationNotCreated()))
+			require.NoError(t, err, "failed while waiting for ToolchainStatus to contain error due to che being required")
+
+			t.Run("verify member and toolchain status go back to ready", func(t *testing.T) {
+				// change che required flag back to true to resolve the error on the memberstatus
+				memberConfigurationWithCheRequired = testconfig.ModifyMemberOperatorConfigObj(memberAwait.GetMemberOperatorConfig(), testconfig.Che().Required(false))
+				hostAwait.UpdateToolchainConfig(testconfig.Members().Default(memberConfigurationWithCheRequired.Spec))
+
+				VerifyMemberStatus(t, memberAwait, consoleURL)
+				VerifyToolchainStatus(t, hostAwait, memberAwait)
+			})
+		})
+	})
+
+	memberAwait.WaitForUsersPodsWebhook()
+	memberAwait.WaitForAutoscalingBufferApp()
+	_, err := hostAwait.WaitForToolchainStatus(wait.UntilToolchainStatusHasConditions(
+		ToolchainStatusReadyAndUnreadyNotificationNotCreated()...))
+	require.NoError(t, err, "failed while waiting for ToolchainStatus")
+
+	rand.Seed(time.Now().UnixNano())
```

Comment:
sure, but shouldn't we wait until the operators and registration services are ready before running the tests?

Reply:
The main checks are done by `WaitForDeployments(t)`. The rest (lines that I commented on) are just extra assertions - it's enough to call them only once - we don't have to replicate it for all tests.

Reply:
ok, fair enough. I wanted to have those checks in place in case we run the `TestSpace` stuff only, but it's not supposed to happen on CI, only on our local machines anyways

Reply:
removed in 101c2fb 🗑️